logging/unit_tests/test__gax.py

```diff
@@ -412,7 +412,7 @@ def test_write_entries_w_extra_properties(self):
             'severity': SEVERITY,
             'labels': LABELS,
             'insertId': IID,
-            'timestamp': NOW,
+            'timestamp': NOW.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
I think you'll have to add a conversion from `datetime` to `string` in the `commit` method from the `Batch` class ([link to code](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/logging/google/cloud/logging/logger.py#L479)).